Makefile

```diff
@@ -3,6 +3,11 @@
 install:
 	pipenv install
 
+environment_variables:
+	export WEBSERVER_PORT=5000
```

Comment:
I don't think this is a good idea to put configuraton values into repo - instead you can have own shell script exporting those variables